portality/lib/query_filters.py

```diff
@@ -186,11 +186,12 @@ def prune_author_emails(results, unpacked=False):
 
 
 def publisher_result_filter(results, unpacked=False):
+    allowed_admin = ["ticked", "seal", "in_doaj", "related_applications", "current_application", "current_journal", "application_status"]
```

Comment:
I've found a test which references these properties, `doajtest/unit/test_query_filters.py` - I'll hotfix that since I've merged this PR already